packages/react/index.ts

```diff
@@ -334,6 +337,7 @@ function addComponent(component: any): void {
  */
 interface ReactSpec extends axeCore.Spec {
   runOnly?: string[];
+  disableCache?: boolean;
```

Comment:
Please describe this option in the readme.

Reply:
Yup, good catch will do :)